doc/source/getting_started/faq.rst

```diff
@@ -12,6 +12,28 @@ Design Language (APDL), Ansys Fluent, and other Ansys products.
 You can use PyAnsys libraries within a Python environment of your choice
 in conjunction with external Python libraries.
 
+What Ansys license do I need to run the Geometry service?
+---------------------------------------------------------
+
+.. note::
+
+   Answered in https://github.com/ansys/pyansys-geometry/discussions/754
+
+The Ansys Geometry service is a headless service developed on top of the
+modelling libraries underneath Discovery and SpaceClaim.
+
+Both in its standalone and dockerized versions, the Ansys Geometry Service
+requires a **Discovery Modeling** license to run.
```

Comment:
```suggestion
Both in its standalone and Docker versions, the Ansys Geometry service
requires a **Discovery Modeling** license to run.
```
I prefer "standalone" as one word, and that is what the wordlist for Google dev doc style guide specifies. (However, a recent Ansys Yammer Grammar posts says that the Ansys style is to hyphenate it. Do we ignore that Ansys post?

Reply:
I prefer the one-word version as well. Let's ignore it for now. It would have to be modified in many other places